website/sidebars.ts

```diff
@@ -177,12 +177,12 @@ const sidebars: SidebarsConfig = {
         { type: 'doc', id: 'integrations/github', label: 'GitHub Action' },
         { type: 'doc', id: 'integrations/jira', label: 'Jira Cloud Plugin' },
         { type: 'doc', id: 'integrations/mixpanel', label: 'Mixpanel' },
-        { type: 'doc', id: 'integrations/segment', label: 'Twilio Segment' },
         { type: 'doc', id: 'integrations/google-analytics', label: 'Google Analytics' },
         { type: 'doc', id: 'integrations/monday', label: 'monday.com' },
         { type: 'doc', id: 'integrations/slack', label: 'Slack' },
         { type: 'doc', id: 'integrations/terraform', label: 'Terraform' },
         { type: 'doc', id: 'integrations/trello', label: 'Trello Power-Up' },
+        { type: 'doc', id: 'integrations/segment', label: 'Twilio Segment' },
```

Comment:
I think the order is wrong here, `google-analytics` should be before `mixpanel` and `segment` should be before `slack`.